tyk-data-plane/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
+name: tyk-data-plane
+description: A Helm chart for deploy a Tyk Multi Data Center Bridge composed on Kubernetes
```

Comment:
Does the description need an update?

Reply:
I will be creating separate PR to update the description of all the charts. Will cover it there.